content/en/docs/concepts/storage/volumes.md

```diff
@@ -142,37 +101,39 @@ spec:
   - name: test-volume
     # This AWS EBS volume must already exist.
     awsElasticBlockStore:
-      volumeID: <volume-id>
+      volumeID: "<volume id>"
       fsType: ext4
 ```
 
-#### CSI Migration
+#### AWS EBS CSI migration
 
 {{< feature-state for_k8s_version="v1.17" state="beta" >}}
 
-The CSI Migration feature for awsElasticBlockStore, when enabled, shims all plugin operations
+The `CSIMigration` feature for `awsElasticBlockStore`, when enabled, shims all plugin operations
 from the existing in-tree plugin to the `ebs.csi.aws.com` Container
-Storage Interface (CSI) Driver. In order to use this feature, the [AWS EBS CSI
-Driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
+Storage Interface (CSI) driver. In order to use this feature, the [AWS EBS CSI
+driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
 must be installed on the cluster and the `CSIMigration` and `CSIMigrationAWS`
-Beta features must be enabled.
+beta features must be enabled.
+
+#### AWS EBS CSI migration complete
 
-#### CSI Migration Complete
 {{< feature-state for_k8s_version="v1.17" state="alpha" >}}
 
-To turn off the awsElasticBlockStore storage plugin from being loaded by controller manager and kubelet, you need to set this feature flag to true. This requires `ebs.csi.aws.com` Container Storage Interface (CSI) driver being installed on all worker nodes.
+To disable the `awsElasticBlockStore` storage plugin from being loaded by the controller manager
+and the kubelet, set the `CSIMigrationAWSComplete` flag to `true`. This feature requires the `ebs.csi.aws.com` Container Storage Interface (CSI) driver installed on all worker nodes.
 
 ### azureDisk {#azuredisk}
```

Comment:
the doc seems to be inconsistent about which volume types are persistent and which are ephemeral. Would it be more clear if we broke up this list of volume types into two sections, ephemeral and persistent?

Reply:
I agree.